hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxySupport.java

```diff
@@ -116,10 +117,13 @@
 abstract class MapProxySupport extends AbstractDistributedObject<MapService> implements InitializingObject {
 
     protected static final String NULL_KEY_IS_NOT_ALLOWED = "Null key is not allowed!";
+
     protected static final String NULL_VALUE_IS_NOT_ALLOWED = "Null value is not allowed!";
     protected static final String NULL_PREDICATE_IS_NOT_ALLOWED = "Predicate should not be null!";
     protected static final String NULL_LISTENER_IS_NOT_ALLOWED = "Null listener is not allowed!";
 
+    private static final int CHECK_IF_LOADED_TIMEOUT = 60;
```

Comment:
Mostly, as far as i see, we are adding unit of timeout at the end like `CHECK_IF_LOADED_TIMEOUT_SECONDS` instead of `CHECK_IF_LOADED_TIMEOUT `. Just a minor thing.